test/e2e/flask/petnames-transactions.spec.js

```diff
@@ -0,0 +1,48 @@
+const {
+  openDapp,
+  switchToNotificationWindow,
+  withFixtures,
+  unlockWallet,
+  defaultGanacheOptions,
+} = require('../helpers');
+const FixtureBuilder = require('../fixture-builder');
+const {
+  expectName,
+  focusTestDapp,
+  rejectSignatureOrTransactionRequest,
+  saveName,
+} = require('./petnames-helpers');
+
+async function createTransactionRequest(driver) {
+  await driver.clickElement('#sendButton');
+  await driver.delay(3000);
+}
+
+describe('Petnames', function () {
```

Comment:
Should we use alternate names here to distinguish the two sets of tests, maybe `Petnames - Signatures` and `Petnames - Transactions`?

Reply:
Done in https://github.com/MetaMask/metamask-extension/pull/22515